apps/fabric-website-resources/src/docs/GettingStartedOverview.md

```diff
@@ -11,18 +11,18 @@ Integrating components into your project depends heavily on your setup. The reco
 Within an npm project, you should install the package and save it as a dependency:
 
 ```bash
-npm install --save office-ui-fabric-react
+npm install --save @fluentui/react
 ```
 
-This will add the fabric-react project as a dependency in your package.json file, and will drop the project under node_modules/office-ui-fabric-react.
+This will add the fabric-react project as a dependency in your package.json file, and will drop the project under node_modules/@fluentui/react.
 
 The library includes commonjs entry points under the lib folder. To use a control, you should be able to import it and use it in your render method. Note that wrapping your application in the Fabric component is required to support RTL, keyboard focus and other features.
```

Comment:
While we're here...this part is wrong. (We should really scrub or remove this file, since I suspect a lot of it is out of date...)
```suggestion
The library includes commonjs entry points under the `lib-commonjs` folder. To use a control, you should be able to import it and use it in your render method. Note that wrapping your application in the Fabric component is required to support RTL, keyboard focus and other features.
```

Also points out the question, what happens to the `Fabric` component?